spring-cloud-dataflow-docs/src/main/asciidoc/howto.adoc

```diff
@@ -82,3 +82,84 @@ environment properties using the platform specific environment-setting capabilit
 in Cloud Foundry, you'd be passing them as `cf set-env SPRING_APPLICATION_JSON`.
 
 
+== Logging
+
+Spring Cloud Data Flow is built upon several Spring projects, but ultimately the dataflow-server is a 
+Spring Boot app, so the logging techniques that applies to any link:http://docs.spring.io/spring-boot/docs/current/reference/html/howto-logging.html#howto-logging[Spring Boot] 
+application is applicable here as well. 
+
+
+While troubleshooting, following are the two primary areas where enabling the DEBUG logs could be 
+useful.
+
+=== Deployment Logs
+Spring Cloud Data Flow builds upon link:https://github.com/spring-cloud/spring-cloud-deployer[Spring Cloud Deployer] SPI 
+and the platform specific dataflow-server uses the respective link:https://github.com/spring-cloud?utf8=%E2%9C%93&query=deployer[SPI implementations]. 
+Specifically, if we were to troubleshoot deployment specific issues; such as the network errors, it'd 
+be useful to enable the DEBUG logs at the deployers and the libraries used by it. 
```

Comment:
We can probably say `the corresponding/underlying deployer` instead of `the deployers` as it comes down to the individual deployer.